pkg/fuzzer/runner.go

```diff
@@ -11,15 +12,22 @@ import (
 	_ "github.com/lib/pq"
 )
 
-// Run the commands in a worker pool
-func Run(fields []drivers.FieldDescriptor, f flags.Flags) error {
+func getDriverAndDB(f flags.Flags) (drivers.Driver, *sql.DB) {
+	driver := drivers.New(f.Driver)
+	db := connector.Connection(driver)
+	db.SetMaxOpenConns(f.Workers)
```

Comment:
If it's set in the connector, don't set it here. Pick one place to set it. I guess this is easier here. But create new flag options for these.